monkey/monkey_island/cc/services/reporting/report.py

```diff
@@ -566,6 +570,18 @@ def get_latest_event_timestamp(cls) -> Optional[float]:
 
         return latest_timestamp
 
+    @classmethod
+    def _get_exploiter_manifests(cls) -> Dict[str, AgentPluginManifest]:
+        exploiter_manifests = cls._agent_plugin_repository.get_all_plugin_manifests().get(  # type: ignore[union-attr] # noqa: E501
+            AgentPluginType.EXPLOITER, {}
+        )
+        if not exploiter_manifests:
+            logger.debug("No plugin exploiter manifests were found")
+
+        exploiter_manifests.update(HARD_CODED_EXPLOITER_MANIFESTS)
```

Comment:
Another one of these statements is added. IMO we should've refactored the repository to include these before this PR